### PR TITLE
Updated breadcrumbs tokens for S2

### PR DESCRIPTION
**Pull request by lynnhao**
## Description

- Updated tokens according to updated token spec for Spectrum 2, including new and deprecated tokens

## Motivation and context

- View detailed context on changes [here](https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?node-id=17961-6037&t=uNkUk2U6k4PeQs7u-4)

## Related issue

[SDS-13413](https://jira.corp.adobe.com/browse/SDS-13413)

## Types of changes

<li> [ ] Patch (bug fixes, typos, mistakes; non-breaking change which fixes an issue) </li>
<li> [ ] Minor (add a new token, changing a value, deprecating a token; non-breaking change which adds functionality) </li>
<li> [x] Major (deleting a token, changing token value type, renaming a token by deprecating the old one; fix or feature that would cause existing functionality to change) </li>

## Checklist:

<li> [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html). </li>
<li> [x] I updated the token in all applicable sets. This applies if updating, adding, or deleting a token that has data across different sets (for example, if the value differs across color themes.) </li>

### CODE REVIEW

**Comment by mrcjhicks** [comment]
**[TESTING spectrum2](https://github.com/adobe/spectrum-tokens-studio-data/actions/runs/9374592040)**

READING spectrum-tokens: `/spectrum-tokens/packages/tokens/src/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-colors/`
READING   tokens studio: `/spectrum-tokens-studio-data/src/tokens-studio/spectrum2-non-colors/`

<details><summary>STUDIO SYNC SUMMARY</summary><p>

```
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-truncated-menu-to-bottom-text
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 breadcrumbs-bottom-to-text
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 breadcrumbs-bottom-to-text-compact
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 breadcrumbs-separator-icon-to-bottom-text-multiline
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 breadcrumbs-start-edge-to-text
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 breadcrumbs-top-to-separator-icon
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 breadcrumbs-top-to-separator-icon-compact
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 breadcrumbs-top-to-text
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 breadcrumbs-top-to-text-compact
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.breadcrumbs-top-to-truncated-menu
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.🚫 breadcrumbs-top-to-truncated-menu-compact
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-separator-to-bottom-text-multiline
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-separator-to-bottom-text-multiline
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-start-edge-to-text-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-start-edge-to-text-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-start-edge-to-text-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-start-edge-to-text-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-start-edge-to-text-multiline
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-start-edge-to-text-multiline
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-top-to-separator-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-top-to-separator-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-top-to-separator-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-top-to-separator-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-top-to-separator-multiline
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-top-to-separator-multiline
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component.breadcrumbs-truncated-menu-to-separator
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-text-to-separator-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-text-to-separator-large
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-text-to-separator-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-text-to-separator-medium
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/desktop.breadcrumbs-text-to-separator-multiline
UPDATED METADATA - SPECTRUM UUID: spectrum2/layout.component/mobile.breadcrumbs-text-to-separator-multiline
```
</p></details>

<details><summary>SPECTRUM SYNC SUMMARY</summary><p>

```
NEW TOKENS: (22) breadcrumbs-truncated-menu-to-bottom-text/mobile, breadcrumbs-separator-to-bottom-text-multiline/desktop, breadcrumbs-separator-to-bottom-text-multiline/mobile, breadcrumbs-start-edge-to-text-large/desktop, breadcrumbs-start-edge-to-text-large/mobile, breadcrumbs-start-edge-to-text-medium/desktop, breadcrumbs-start-edge-to-text-medium/mobile, breadcrumbs-start-edge-to-text-multiline/desktop, breadcrumbs-start-edge-to-text-multiline/mobile, breadcrumbs-top-to-separator-large/desktop, breadcrumbs-top-to-separator-large/mobile, breadcrumbs-top-to-separator-medium/desktop, breadcrumbs-top-to-separator-medium/mobile, breadcrumbs-top-to-separator-multiline/desktop, breadcrumbs-top-to-separator-multiline/mobile, breadcrumbs-truncated-menu-to-separator, breadcrumbs-text-to-separator-large/desktop, breadcrumbs-text-to-separator-large/mobile, breadcrumbs-text-to-separator-medium/desktop, breadcrumbs-text-to-separator-medium/mobile, breadcrumbs-text-to-separator-multiline/desktop, breadcrumbs-text-to-separator-multiline/mobile
MODIFIED TOKENS: (11) breadcrumbs-truncated-menu-to-bottom-text/desktop, breadcrumbs-bottom-to-text, breadcrumbs-bottom-to-text-compact, breadcrumbs-separator-icon-to-bottom-text-multiline, breadcrumbs-start-edge-to-text, breadcrumbs-top-to-separator-icon, breadcrumbs-top-to-separator-icon-compact, breadcrumbs-top-to-text, breadcrumbs-top-to-text-compact, breadcrumbs-top-to-truncated-menu, breadcrumbs-top-to-truncated-menu-compact
DEPRECATED TOKENS: (24) breadcrumbs-height, breadcrumbs-height-compact, breadcrumbs-top-to-text/desktop, breadcrumbs-top-to-text/mobile, breadcrumbs-top-to-text-compact/desktop, breadcrumbs-top-to-text-compact/mobile, breadcrumbs-bottom-to-text/desktop, breadcrumbs-bottom-to-text/mobile, breadcrumbs-bottom-to-text-compact/desktop, breadcrumbs-bottom-to-text-compact/mobile, breadcrumbs-start-edge-to-text/desktop, breadcrumbs-start-edge-to-text/mobile, breadcrumbs-top-to-separator-icon/desktop, breadcrumbs-top-to-separator-icon/mobile, breadcrumbs-top-to-separator-icon-compact/desktop, breadcrumbs-top-to-separator-icon-compact/mobile, breadcrumbs-top-to-separator-icon-multiline/desktop, breadcrumbs-top-to-separator-icon-multiline/mobile, breadcrumbs-separator-icon-to-bottom-text-multiline/desktop, breadcrumbs-separator-icon-to-bottom-text-multiline/mobile, breadcrumbs-truncated-menu-to-separator-icon, breadcrumbs-top-to-truncated-menu/mobile, breadcrumbs-top-to-truncated-menu-compact/desktop, breadcrumbs-top-to-truncated-menu-compact/mobile
CHANGED VALUES: (20) breadcrumbs-top-to-separator-icon-multiline/desktop, breadcrumbs-top-to-text-multiline/desktop, breadcrumbs-bottom-to-text-multiline/mobile, breadcrumbs-height-multiline/mobile, breadcrumbs-top-to-separator-icon-multiline/mobile, breadcrumbs-top-to-text-multiline/mobile, breadcrumbs-height, breadcrumbs-height-compact, breadcrumbs-truncated-menu-to-separator-icon, breadcrumbs-truncated-menu-to-bottom-text, breadcrumbs-bottom-to-text/desktop, breadcrumbs-bottom-to-text-compact/desktop, breadcrumbs-separator-icon-to-bottom-text-multiline/desktop, breadcrumbs-start-edge-to-text/desktop, breadcrumbs-top-to-separator-icon/desktop, breadcrumbs-top-to-separator-icon-compact/desktop, breadcrumbs-top-to-text/desktop, breadcrumbs-top-to-text-compact/desktop, breadcrumbs-top-to-truncated-menu/desktop, breadcrumbs-top-to-truncated-menu-compact/desktop impacting 18 tokens
CHANGED SCHEMAS: (11) breadcrumbs-top-to-separator-icon-multiline, breadcrumbs-truncated-menu-to-separator-icon, breadcrumbs-bottom-to-text, breadcrumbs-bottom-to-text-compact, breadcrumbs-separator-icon-to-bottom-text-multiline, breadcrumbs-start-edge-to-text, breadcrumbs-top-to-separator-icon, breadcrumbs-top-to-separator-icon-compact, breadcrumbs-top-to-text, breadcrumbs-top-to-text-compact, breadcrumbs-top-to-truncated-menu-compact
```
</p></details>

TEST COMPLETE

**Review by nabuhasan — APPROVED** [review]
Everything looks good to me!

File: src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json [hunk 1]
```diff
@@ -779,8 +779,8 @@
       }
     }
   },
-  "breadcrumbs-height-compact": {
-    "value": "{component-height-200}",
+  "🚫 breadcrumbs-height-compact": {
+    "value": "{component-height-100}",
```
Comment (karstens) [inline src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json:783]: @GarthDB @lynnhao @nabuhasan I really wonder, if it is okay, that we deprecate a token and change the value in one step? Doesn't that lead to a breaking change?

**Review by karstens — COMMENTED** [review]
Looks good to me, just the general question, if we can change the token value in the same step as deprecating it? cc @misterbrownlee

**Comment by GarthDB** [comment]
> Looks good to me, just the general question, if we can change the token value in the same step as deprecating it? cc @misterbrownlee

Yes, it is recommended. When we rename tokens, we should create a token with the new name, deprecate the old token, and point it to the new token name.

**Review by GarthDB — APPROVED** [review]

**Comment by GarthDB** [comment]
@lynnhao for the future, I don't think a link to figma provides enough context for implementations for design motivation. I use this information in my release notes, and a screen shot would be better, but a text description would be best.

**Comment by lynnhao** [comment]
> @lynnhao for the future, I don't think a link to figma provides enough context for implementations for design motivation. I use this information in my release notes, and a screen shot would be better, but a text description would be best.

Okay got it! Good to know you use them directly.

**Comment by karstens** [comment]
> > Looks good to me, just the general question, if we can change the token value in the same step as deprecating it? cc @misterbrownlee
> 
> Yes, it is recommended. When we rename tokens, we should create a token with the new name, deprecate the old token, and point it to the new token name.

@GarthDB Thank you for your input. Please look at the code / my comment: the token is deprecated, but the value changes from `{component-height-200}` to `{component-height-100}`. So, the deprected token does not point to the new token, but the value changes. Sorry, I don't understand how that can be recommended?! cc @misterbrownlee

**Pull request merged**